<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>761</width>
     <height>561</height>
    </rect>
   </property>
   <widget class="QTabWidget" name="Generate">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>761</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_generate">
     <attribute name="title">
      <string>Generate</string>
     </attribute>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>0</y>
        <width>581</width>
        <height>451</height>
       </rect>
      </property>
      <widget class="QScrollArea" name="scrollArea">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>10</y>
         <width>161</width>
         <height>431</height>
        </rect>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>159</width>
          <height>429</height>
         </rect>
        </property>
        <widget class="QScrollArea" name="scrollArea_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>141</width>
           <height>411</height>
          </rect>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_4">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>139</width>
            <height>409</height>
           </rect>
          </property>
          <widget class="QScrollBar" name="original_view_scroll">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>0</y>
             <width>21</width>
             <height>411</height>
            </rect>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>121</width>
             <height>411</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>image.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </widget>
        </widget>
       </widget>
      </widget>
      <widget class="QGraphicsView" name="graphicsView">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>391</width>
         <height>431</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>321</width>
         <height>411</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>manual.jpg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="open_in_editor_button">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>450</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>open in editor</string>
      </property>
     </widget>
     <widget class="QTreeView" name="treeView_generated">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>151</width>
        <height>431</height>
       </rect>
      </property>
     </widget>
     <widget class="QTreeWidget" name="treeWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>131</width>
        <height>411</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>mechanic_parts</string>
       </property>
      </column>
      <item>
       <property name="text">
        <string>YBB100</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>YBB200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>YBB200-124</string>
       </property>
       <item>
        <property name="text">
         <string>YBB202</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>YBB203</string>
        </property>
       </item>
      </item>
      <item>
       <property name="text">
        <string>YBB133</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>YYB114</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>YYB1919</string>
       </property>
       <item>
        <property name="text">
         <string>YBB1919-114514</string>
        </property>
       </item>
      </item>
     </widget>
     <widget class="QPushButton" name="generate_button">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>450</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Generate</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_modify">
     <attribute name="title">
      <string>Editor</string>
     </attribute>
     <widget class="QWidget" name="widget_3" native="true">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>581</width>
        <height>451</height>
       </rect>
      </property>
      <widget class="QScrollArea" name="scrollArea_2">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>0</y>
         <width>161</width>
         <height>441</height>
        </rect>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>159</width>
          <height>439</height>
         </rect>
        </property>
        <widget class="QScrollBar" name="original_view_scroll_2">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>10</y>
           <width>20</width>
           <height>421</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
        <widget class="QTreeView" name="treeView_original_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>121</width>
           <height>381</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>400</y>
           <width>121</width>
           <height>22</height>
          </rect>
         </property>
         <property name="whatsThis">
          <string extracomment="Search">Search</string>
         </property>
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="maxLength">
          <number>32767</number>
         </property>
         <property name="placeholderText">
          <string>Search</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>121</width>
           <height>381</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>image.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QQuickWidget" name="quickWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>401</width>
         <height>451</height>
        </rect>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="resizeMode">
        <enum>QQuickWidget::SizeRootObjectToView</enum>
       </property>
      </widget>
     </widget>
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>151</width>
        <height>441</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Annotation</string>
       </attribute>
       <widget class="QTextEdit" name="textEdit">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>170</y>
          <width>141</width>
          <height>201</height>
         </rect>
        </property>
       </widget>
       <widget class="QScrollArea" name="scrollArea_5">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>141</width>
          <height>161</height>
         </rect>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_6">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>139</width>
           <height>159</height>
          </rect>
         </property>
         <widget class="QScrollBar" name="verticalScrollBar_2">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>0</y>
            <width>16</width>
            <height>211</height>
           </rect>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
         <widget class="QCheckBox" name="checkBox_4">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>91</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Part1</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="checkBox_5">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>91</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Part2</string>
          </property>
          <property name="tristate">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QCheckBox" name="checkBox_6">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>70</y>
            <width>91</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Part3</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="checkBox_7">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>100</y>
            <width>91</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Part4</string>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>380</y>
          <width>71</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>save</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>380</y>
          <width>71</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>preview</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="generate_button_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>450</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Generate</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>470</y>
        <width>281</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>*hint: 'wsad' and mouse to move and zoom the view</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_save">
     <attribute name="title">
      <string>Save</string>
     </attribute>
     <widget class="QPushButton" name="save_cancel">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>460</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>cancel</string>
      </property>
     </widget>
     <widget class="QWidget" name="widget_4" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>401</width>
        <height>451</height>
       </rect>
      </property>
      <widget class="QGraphicsView" name="graphicsView_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>391</width>
         <height>431</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>321</width>
         <height>411</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>manual.jpg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
     <widget class="QTextEdit" name="textEditSaveInfo">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>10</y>
        <width>341</width>
        <height>381</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;ID: 1&lt;br /&gt;Size: 348KB&lt;br /&gt;Pages: 1&lt;br /&gt;Container: PDF&lt;br /&gt;Format profile: High@L4&lt;br /&gt;Page size: 21.587 x 27.937&lt;br /&gt;Color space: YUV&lt;br /&gt;Stream size: 4.34 GiB (97%)&lt;br /&gt;Writing library: core 157 r2969 d4099dd&lt;br /&gt;&lt;br /&gt;Author information:&lt;br /&gt;Date: dd/mm/yyyy&lt;br /&gt;Security: None&lt;br /&gt;Read only: No&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="save_save">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>460</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>save</string>
      </property>
     </widget>
     <widget class="QToolButton" name="openFolder">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>400</y>
        <width>21</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>400</y>
        <width>321</width>
        <height>41</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>default C:/system/Appdata/Roam/CADM/doc</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>ViewManual</string>
     </attribute>
     <widget class="QTextEdit" name="textEdit_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>611</width>
        <height>481</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Save as PDF</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Update Annotation</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Image</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="feedback_button">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>0</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Feedback</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
